docs/install_guides/troubleshooting-steps.md

```diff
@@ -0,0 +1,84 @@
+
+@sphinxdirective
+
+.. tab:: Linux and macOS
+
+   .. code-block:: sh
+   
+      pip install --upgrade pip
+   
+.. tab:: Windows
+
+   .. code-block:: sh
+   
+      python -m pip install --upgrade pip
+
+
+@endsphinxdirective
+
+
+<!--## Check the special tips for Anaconda installation-->
+
+<!--missing part-->
+
+
+## Check if required external dependencies is installed
+
+For Ubuntu and RHEL 8 systems, if you installed OpenVINO Runtime via the installer, APT, or YUM repository, and decided to [install OpenVINO Development Tools](installing-model-dev-tools.md), make sure that you <a href="openvino_docs_install_guides_installing_openvino_linux.html#install-external-dependencies">Install External Software Dependencies</a> first. 
+
+For Windows systems, if C++ is still required, make sure that Microsoft Visual Studio 2019 with MSBuild and CMake 3.14 or higher (64-bit) are installed. While installing Microsoft Visual Studio 2019, make sure that you have selected **Desktop development with C++** in the **Workloads** tab. If not, launch the installer again to select that option. For more information on modifying the installation options for Microsoft Visual Studio, see its [official support page](https://docs.microsoft.com/en-us/visualstudio/install/modify-visual-studio?view=vs-2022).
+
+## Check if you have installed OpenVINO before 
+
+If you have installed OpenVINO before and added `setupvars` to your `PATH /.bashrc`, after restarting your environment, you should see the following information:
+```sh
+[setupvars.sh] OpenVINO™ environment initialized
+```
+
+- If you don't see similar information, your PATH variable may be configured incorrectly. Check if you have written the correct <INSTALL_DIR> or tried to activate it in the right folder.
+- If you added it to a `.bashrc` file, make sure that the command is correctly written and the file is found in the `~/.bashrc` folder.
+
+## Verify if OpenVINO is correctly installed
+
+To verify if OpenVINO is correctly installed, use the following command:
+```sh
+python -c "from openvino.runtime import Core"
+```
+
+If OpenVINO was successfully installed, nothing will happen. If not, an error will be displayed. 
+
+## Check if GPU drvier is installed
+
+[Additional configurations](configurations-header.md) are required in order to use OpenVINO on different hardware.
+
+To run inference on GPU, make sure that you have installed the correct GPU driver. To check that, see [additional configurations for GPU](configurations-for-intel-gpu.md).
+
+## Firewall/Network Issues
+
+Make sure that your firewall and network settings are working as expected. For issues caused by this reason, see <a href="openvino_docs_get_started_guide_troubleshooting_issues.html#install-for-prc">Errors with Installing via PIP for PRC Users</a> and <a href="openvino_docs_get_started_guide_troubleshooting_issues.html#proxy-issues">proxy issues with installing OpenVINO on Linux from Docker</a>.
```

Comment:
You can also have proxy issues apart from the specific docker and PRC contexts, so I would rephrase this as just two examples

Reply:
Actually the two examples are specific: the PRC one uses aliyun as a mirror repo (because PRC users constantly have this proxy issue), and the proxy example is directing users to a section about how to set proxy for Docker installation.

Reply:
Understood, but this phrasing indicates that issues caused by "firewall and network settings" are addressed in the PRC and Docker sections. People who are not in PRC and do not use Docker can also have issues with network settings (for example: "pip install" will not work if I am on VPN without configuring proxy. Usually people will probably want to set proxy settings system wide because many commands fail if you do not, but for installation issues, it might be simplest to specify options for using pip behind a proxy, like `pip install --proxy http://address:port --trusted-host pypi.org openvino`. That would allow at least installation, but a command like `omz_downloader` would still fail if generic proxy settings are not set up.

Reply:
I see. Let me rephrase that.

Reply:
Updated. Could you help check again?